force_nevergrad/engine/nevergrad_optimizers.py

```diff
@@ -71,7 +72,7 @@ def nevergrad_function(*ng_params,
     mco_params = translate_ng_to_mco(list(ng_params))
 
     # call the MCO objective function
-    objective = function(mco_params)
+    objective = deepcopy(function(mco_params))
```

Comment:
Is there a reason why we need to perform this deep copy? If it's only to get the unit tests working then we should probably review this.

Additionally the test that fails looks like it's related to the `minimize_objectives` argument in the `nevergrad_function` function. It's worth pointing out that we shouldn't need to include this once https://github.com/force-h2020/force-bdss/pull/353 is merged, since all the `KPISpecification.objective` mappings for the objective function are taken care of in the `BaseOptimizerEngine` class

Reply:
I have a solution to the code coverage ... working on it.